app/javascript/packages/normalize-yaml/index.spec.js

```diff
@@ -0,0 +1,32 @@
+import normalize from './index.js';
```

Comment:
should we remove either this file or `spec/javascripts/packages/normalize-yaml/index-spec.js`?

Reply:
> should we remove either this file or `spec/javascripts/packages/normalize-yaml/index-spec.js`?

Whoops, yes.

For context, I've historically waffled between putting specs in `specs/` vs. co-located with the package, and had thought to consolidate to `specs/` for consistency. But as seen with the [build failure](https://app.circleci.com/pipelines/github/18F/identity-idp/38863/workflows/03bf0965-fb5e-4950-8add-decb34aa6cbb/jobs/63989), there's an incompatibility with how JavaScript files in `specs/` are transpiled with Babel (i.e. ESM -> CommonJS) and the package itself being `"type": "module"`. So, co-located it is!

Reply:
Removed the `spec/` file in 4c127db.